### ISSUE TITLE
Unable to manually update cruft inside devcontainer - AZ is not installed

### ISSUE THREAD

**Issue by kjaymiller** [kjaymiller/cookiecutter-relecloud#280]
One of our cruft hooks uses az bicep config and Azure CLI is not installed in the devcontainer

**Comment by pamelafox** [comment]
I don't think we should generally be adding a lot of tools to dev container that are just for cruft update, but I guess azure CLI is generally useful enough to our customers that it's a good fit.
